R/helper.R

```diff
@@ -0,0 +1,43 @@
+#' Retrieve SingleR reference datasets from Github
+#'
+#' \code{getSinglerRef} retrieves \code{SingleR} reference datasets from Github. 
+#'
+#' @param dataset String referring to SingleR reference dataset to retrieve. 
```

Comment:
There's no need to explicitly list the options here, if they are implicitly present in the argument list of the function (see my other comment). I would recommend moving a full description of the options into `@details` along with some plain-English details about what each one is. This information is probably in the old SingleR repo.